src/lib/utils/common-helper.ts

```diff
@@ -77,4 +78,90 @@ function pluralize(count: number, singular: string, plural: string): string {
     return count === 1 ? singular : plural;
 }
 
-export { getName, getNames, isPaperUndecided, doesPaperNeedReview, exhaustiveCheck, pluralize };
+/**
+ * Groups a list by keys.
+ * The keys are indirectly given by the keySelector function.
+ *
+ * Inspired by: https://stackoverflow.com/questions/14446511/most-efficient-method-to-groupby-on-an-array-of-objects
+ *
+ * @param list the list to be grouped
+ * @param keySelector function that map a (list) item to a certain key
+ * @returns the grouped list as an object with the association key: \<list of items associated to key\>
+ */
+function groupBy<T>(list: T[], keySelector: (arg0: T) => string): Record<string, T[]> {
+    return list.reduce((result: Record<string, T[]>, item: T) => {
+        (result[keySelector(item)] ??= []).push(item);
+        return result;
+    }, {});
+}
+
+type colorPrefix = "" | "text" | "bg" | "fill";
+const statusColors: Record<PaperStatus, Record<colorPrefix, string>> = {
+    Accepted: {
+        "": "accept-green",
+        text: "text-accept-green",
+        fill: "fill-accept-green",
+        bg: "bg-accept-green",
+    },
+    Declined: {
+        "": "decline-red",
+        text: "text-decline-red",
+        fill: "fill-decline-red",
+        bg: "bg-decline-red",
+    },
+    Undecided: {
+        "": "maybe-yellow",
+        text: "text-maybe-yellow",
+        fill: "fill-maybe-yellow",
+        bg: "bg-maybe-yellow",
+    },
+    "Not reviewed": {
+        "": "unreviewed-gray",
+        text: "text-unreviewed-gray",
+        fill: "fill-unreviewed-gray",
+        bg: "bg-unreviewed-gray",
+    },
+};
```

Comment:
There's no class without any prefix
```suggestion
type colorPrefix = "text" | "bg" | "fill";
const statusColors: Record<PaperStatus, Record<colorPrefix, string>> = {
    Accepted: {
        text: "text-accept-green",
        fill: "fill-accept-green",
        bg: "bg-accept-green",
    },
    Declined: {
        text: "text-decline-red",
        fill: "fill-decline-red",
        bg: "bg-decline-red",
    },
    Undecided: {
        text: "text-maybe-yellow",
        fill: "fill-maybe-yellow",
        bg: "bg-maybe-yellow",
    },
    "Not reviewed": {
        text: "text-unreviewed-gray",
        fill: "fill-unreviewed-gray",
        bg: "bg-unreviewed-gray",
    },
};
```

Reply:
In `PaperListEntry` we use a very similar logic in `getReviewDecisionColor`. Please replace it with `getStatusColor(getStatusText(paper), "border"`.
We probably could also use this for the decision buttons, but changing these would go too far.

Reply:
Good catch.

And regarding the decision buttons. You are right, maybe this could be reused, but we can also do this when implementing #52.